frame/dapps-staking/src/pallet/mod.rs

```diff
@@ -180,6 +180,20 @@ pub mod pallet {
         ValueQuery,
     >;
 
+    /// To store the reward delegates.
+    // Using a separate RewardDelegate storage item allow to store 
+    // the delegate info once for each (reward owner, contract) pair,
+    // which is more storage-efficient. It also keeps modularity.
+    #[pallet::storage]
+    pub type RewardDelegate<T: Config> = StorageDoubleMap<
+        _,
+        Twox64Concat,
```

Comment:
I was thinking about performance when I choose it. But giving it a second view since the user can kind of 
arbitrarily choose the account ID that will be used as index It should be changed to Blake2_128Concat.